c7n/resources/rds.py

```diff
@@ -281,6 +282,107 @@ def process(self, dbs):
                 **params)
 
 
+@filters.register('upgrade-available')
+class UpgradeAvailable(Filter):
+
+    schema = type_schema('upgrade-available')
```

Comment:
we do want a value: true|false option here for the unmark step in a multi-policy workflow.